datastore/h2gis/src/test/groovy/org/orbisgis/datastore/h2gis/H2GISTest.groovy

```diff
@@ -59,4 +62,67 @@ class H2GISTest {
         assert 100 == ds.fetchSize
         assert 50 == ds.batchInsertSize
     }
+
+    @Test
+    void openH2GISFromPath() {
+        def ds = H2GIS.open("./target/db_h2gis")
+        assert ds
+    }
+
+    @Test
+    void createReadFeatureSource() {
+        def ds = H2GIS.open("./target/db_h2gis")
+        assert ds
+        ds.execute("""
+                DROP TABLE IF EXISTS geotable;
+                CREATE TABLE geotable (id int, the_geom geometry(point));
+                INSERT INTO geotable VALUES (1, 'POINT(10 10)'::GEOMETRY), (2, 'POINT(1 1)'::GEOMETRY);
+        """)
+        def fs = ds.getFeatureSource("GEOTABLE")
+        assert fs
+        assert fs.getCount(Query.ALL)==2
+    }
+
+    @Test
+    void eachRowSQL() {
+        def ds = H2GIS.open("./target/db_h2gis")
+        assert ds
+        ds.execute("""
+                DROP TABLE IF EXISTS geotable;
+                CREATE TABLE geotable (id int, the_geom geometry(point));
+                INSERT INTO geotable VALUES (1, 'POINT(10 10)'::GEOMETRY), (2, 'POINT(1 1)'::GEOMETRY);
+        """)
+        def concat = ""
+        ds.eachRow "SELECT THE_GEOM FROM geotable", { row -> concat += "$row.the_geom\n" }
+        assertEquals("POINT (10 10)\nPOINT (1 1)\n", concat)
+    }
+
+    @Test
+    void firstRows() {
+        def ds = H2GIS.open("./target/db_h2gis")
+        assert ds
+        ds.execute("""
+                DROP TABLE IF EXISTS geotable;
+                CREATE TABLE geotable (id int, the_geom geometry(point));
+                INSERT INTO geotable VALUES (1, 'POINT(10 10)'::GEOMETRY), (2, 'POINT(1 1)'::GEOMETRY);
+        """)
+        def concat = ""
+        println ds.firstRow("select count(*) as nb from geotable").nb
+    }
+
+    @Test
+    void queryH2GISMetaData() {
+        def ds = H2GIS.open("./target/db_h2gis")
+        assert ds
+        ds.execute("""
+                DROP TABLE IF EXISTS geotable;
+                CREATE TABLE geotable (id int, the_geom geometry(point));
+                INSERT INTO geotable VALUES (1, 'POINT(10 10)'::GEOMETRY), (2, 'POINT(1 1)'::GEOMETRY);
+        """)
+        def concat = ""
+        ds.rows "SELECT * FROM geotable", { meta ->
+            concat += "${meta.getTableName(1)} $meta.columnCount\n"
+        }
+        assertEquals("GEOTABLE 2\n", concat)
+    }
```

Comment:
Those tests are duplication of test written in the file [JDBCDataStoreUtilsTest](https://github.com/orbisgis/orbisgis/blob/dev/datastore/jdbc-utils/src/test/groovy/org/orbisgis/datastore/jdbcutils/JDBCDataStoreUtilsTest.groovy)